sync2jira/downstream_issue.py

```diff
@@ -947,6 +954,24 @@ def _update_jira_labels(issue, labels):
     issue.update(data)
     log.info('Updated %s tag(s)' % len(_labels))
 
+def _update_github_project_fields(client, existing, issue, github_project_fields):
+    """Update a Jira issue with GitHub project item field values
+
+    :param jira.client.JIRA client: JIRA client
+    :param jira.resource.Issue existing: Existing JIRA issue
+    :param sync2jira.intermediary.Issue issue: Upstream issue
+    :param list: Fields representing GitHub project item fields in GitHub and Jira
+    """
+    github_project_fields = issue.downstream.get('github_project_fields', {})
+
```

Comment:
This assignment overwrites the parameter value.  Assuming that we fetched the appropriate value at line 738 and passed it into this function, we can omit this operation from here.

However, note that the fallback value at line 738 is an empty list while the fallback value here is an empty dictionary.  (I think we want the fallback from here but we want to make the call at line 738.)

Reply:
It was a regression that happened after changing the data structure. Fixed in 46248b8bbea26b8508001cedd23126f6719599b9